docs/calimero_sdk/2_wallet.mdx

```diff
@@ -2,32 +2,34 @@
 title: Wallet SDK
 sidebar_position: 2
 ---
-The SDK enables the DApp to connect to a private shard through the NEAR wallet.
+
+The Wallet SDK enables your DApp to connect to Calimero's private shard through the NEAR wallet. Follow the steps below to interact with the Calimero SDK and integrate it into your project.
 
 ## Interacting with the SDK
 
 To interact with the Calimero SDK, the DApp is developed using Next.js and relies on the Calimero SDK, which is a Node.js dependency available on npmjs. 
 
 ## Initialize the SDK
 
-- To initialize the Calimero SDK, the authentication token obtained from the Calimero console is inputted, along with the RPC endpoint. 
+- To initialize the Calimero SDK, the authentication token obtained from the Calimero console is inputted, along with the RPC endpoint
 
-- The DApp then invokes functions like `requestSignIn` or `requestSignTransaction`, which redirect users to the NEAR wallet for approval. 
+- The DApp then invokes functions like `requestSignIn` or `requestSignTransaction`, which redirect users to the NEAR wallet for approval 
 
-- Upon approval of the requested action, such as sign-in, the connection to the Calimero shard is established.
+- Upon approval of the requested action, such as sign-in, the connection to the Calimero shard is established
 
 For first-time sign-ins:
 
-- The sync API is called within the Calimero console wallet API service. 
+- The sync API is called within the Calimero console wallet API service
```

Comment:
sync API is called from the wallet , Dapp -> Wallet -> Calimero Console -> Calimero gateway -> Calimero wallet service